src/main/java/com/eerussianguy/blazemap/util/Profilers.java

```diff
@@ -13,10 +13,8 @@ public static class Server {
         public static final TimeProfilerAsync TRANSFORMER_TIME_PROFILER = new TimeProfilerAsync(20);
         public static final LoadProfiler TRANSFORMER_LOAD_PROFILER = new LoadProfiler(20, 50);
 
-        public static final TimeProfilerSync CHUNKHOLDER_TIME_PROFILER = new TimeProfilerSync(20);
-        public static final LoadProfiler CHUNKHOLDER_LOAD_PROFILER = new LoadProfiler(20, 50);
-        public static final TimeProfilerSync SERVERLEVEL_TIME_PROFILER = new TimeProfilerSync(20);
-        public static final LoadProfiler SERVERLEVEL_LOAD_PROFILER = new LoadProfiler(20, 50);
+        public static final TimeProfilerSync TRIGGER_CHUNK_DIRTY_TIME_PROFILER = new TimeProfilerSync(20);
+        public static final LoadProfiler TRIGGER_CHUNK_DIRTY_LOAD_PROFILER = new LoadProfiler(20, 50);
```

Comment:
Please undo these (and correct its references) as I decided to keep the naming and I'll be reusing the ServerLevel ones for a different thing.